demos/ensembles/ensemble-link-prediction-example.ipynb

```diff
@@ -1146,6 +1146,15 @@
     "![image.png](attachment:image.png)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "\n",
+    "In this example, we used `stellargraph`s `BaggingEnsemble` class of GraphSAGE models to predict citation links in the Cora dataset. The `BaggingEnsemble` class brings ensemble learning to `stellargraph`'s graph neural network models, e.g., `GraphSAGE`, quantifying prediction variance and potentially improving prediction accuracy. "
```

Comment:
The formatting here doesn't render well on rtd:

<img width="731" alt="Screen Shot 2020-05-05 at 10 51 35 am" src="https://user-images.githubusercontent.com/4321744/81026735-6f45d780-8ebe-11ea-9bfe-814bd9a17aba.png">

Reply:
Could you be more specific about what you mean?

That said, I think this can be removed now after https://github.com/stellargraph/stellargraph/pull/1460, there's no longer the block quote that hit spatialaudio/nbsphinx#450, and it works again: https://stellargraph.readthedocs.io/en/latest/demos/ensembles/ensemble-link-prediction-example.html (see the end)

Reply:
For specifics, I meant "stellargraph" there doesn't work well with the (inconsistent) `'s` and `s`.  This PR is now images only though